projects/baby_driver/scripts/test_adc_read.py

```diff
@@ -0,0 +1,49 @@
+"""This Module Tests methods in adc_read.py"""
+import unittest
+from unittest.mock import patch
+
+from adc_read import adc_read, NUM_PINS_PER_PORT
+from can_util import Message
+from gpio_port import GpioPort
+
+RAW_MAX = 4095
+CONVERTED_MAX = 3000
+
+class TestADCRead(unittest.TestCase):
+    """Test Babydriver's adc_read function"""
+
+    @patch('can_util.send_message')
+    @patch('can_util.next_message')
+    def test_check_range_pin(self):
```

Comment:
Ah, it turns out I lied: this test is failing because `unittest` is trying to pass it `mock_next_message` and `mock_send_message` as parameters, but the function doesn't take them. You'll have to add them to the parameter list.